### PR TITLE
EYZ-286 Add iOS inapp purchase native extensions

### PR DESCRIPTION
**Pull request by dseeker**
extend android methods

### CODE REVIEW

File: android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java [hunk 1]
```diff
@@ -20,9 +25,9 @@
     private static final String TAG = "WeexPluginInappModule";
 
     // Define the callback
-    JSCallback jsCallback;
-    Activity thisActivity;
-    String productId;
+    private JSCallback jsCallback;
```
Comment (phanisaripalli) [inline android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java]: thanks for this :)

**Comment by dseeker** [comment]
@phanisaripalli I don't know how the formatting for Java should look, go ahead and do the cosmetic touches you think should be there and commit

**Comment by phanisaripalli** [comment]
@dseeker committed and pushed

File: android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java [hunk 2]
```diff
@@ -96,17 +100,21 @@ public void manageSubscriptions() {
...
+        Uri webpage = Uri.parse("http://play.google.com/store/account/subscriptions");
+        Intent intent = new Intent(Intent.ACTION_VIEW, webpage);
+        this.thisActivity.startActivity(intent);
     }
 
     @JSMethod(uiThread = true)
     public void getProductInfo(String params, JSCallback jsCallback) {
         Log.d(TAG, "-> getProductInfo");
         JSONObject json = new JSONObject();
-        try { json = new JSONObject(params); } catch (Throwable t) { Log.e("-> show", "Could not parse malformed JSON: \"" + json + "\""); }
```
Comment (eyzhub) [inline android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java]: @phanisaripalli is there any way we can refactor this to become a method and call it from both methods that are using this parsing?

File: android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java [hunk 3]
```diff
@@ -49,11 +49,15 @@ public void asyncRet(String param, JSCallback callback) {
     @JSMethod(uiThread = true)
     public void show(String params) throws JSONException {
         JSONObject json = new JSONObject();
-        try { json = new JSONObject(params); } catch (Throwable t) { Log.e("-> show", "Could not parse malformed JSON: \"" + json + "\""); }
```
Comment (dseeker) [inline android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java]: @phanisaripalli is there any way we can refactor this to become a method and call it from both methods that are using this parsing?

Comment (phanisaripalli) [reply android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java]: yea, sure. let me commit.

Comment (phanisaripalli) [reply android/library/src/main/java/org/weex/plugin/weexplugininapp/WeexPluginInappModule.java]: @dseeker please have a look

**Pull request merged**